src/Type/PostObject/Connection/PostObjectConnectionArgs.php

```diff
@@ -51,12 +51,12 @@ class PostObjectConnectionArgs extends WPInputObjectType {
 
 	/**
 	 * PostObjectConnectionArgs constructor.
-	 *
+	 * @param string $connection
```

Comment:
Should also probably have a `@param` for `$config` here as well. Some details here would also be helpful.